crates/speedy-transform/src/web_transform/babel_import.rs

```diff
@@ -54,20 +58,41 @@ pub fn transform_style(
         for specifier in &var.specifiers {
           match specifier {
             ImportSpecifier::Named(ref s) => {
-              let ident: String = s.local.sym.to_string();
+              let imported = match &s.imported {
+                Some(imported) => match &imported {
+                  ModuleExportName::Ident(ident) => Option::Some(ident.sym.to_string()),
+                  ModuleExportName::Str(ident) => Option::Some(ident.value.to_string()),
+                },
+                None => Option::None,
+              };
+              // 当 imported 不为 none 时, local.sym 是引入组件的 as 别名
+              let as_name: Option<String> = if imported.is_some() {
+                Option::Some(s.local.sym.to_string())
+              } else {
+                Option::None
+              };
+              // 当 imported 不为 none 时, 实际引入的组件命名为 imported, 否则为 s.local.sym
+              let ident: String = imported.unwrap_or(s.local.sym.to_string());
+
               if match_ident(&s.local) {
                 // 替换对应的 css
                 if let Some(ref css) = child_config.replace_css {
                   let ignore_component = &css.ignore_style_component;
                   let need_lower = css.lower.unwrap_or(false);
-                  let css_ident = if need_lower {
-                    ident.to_lowercase()
-                  } else {
-                    ident.clone()
+                  let camel2dash = css.camel2_dash_component_name.unwrap_or(true);
```

Comment:
I think this will cause a breaking change

Reply:
I make this option default value to false, so that it won't cause a breaking change.

Reply:
LGTM